### DISCUSSION TITLE
Currently, the official packages provided by the community do not include client Mapreduce packages.

### DISCUSSION THREAD

**Discussion by yanfangwang** [apache/celeborn#2500] [Q&A]
<img width="958" alt="image" src="https://github.com/apache/celeborn/assets/169323179/0d4a21db-f2bb-4dd0-a13d-8c4b9eaa9d39">
下边是自己打包出来带MR的，相比较一下社区的包和我编译的有无区别
![image](https://github.com/apache/celeborn/assets/169323179/a48dcc07-3a70-4064-b7db-51ab4b0263d4)

**Comment by FMX** [comment]
<img width="829" alt="截屏2024-05-10 17 32 43" src="https://github.com/apache/celeborn/assets/4150993/9d0a59d9-f944-4f9b-84dd-fa5945377405">

上图中的 mr 目录就是 celeborn release 的 mr client。
